### PR TITLE
dev-libs/device-atlas-api-c: bump to version 2.1.4

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

File: dev-libs/device-atlas-api-c/device-atlas-api-c-2.1.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+multilib_src_install_all() {
+	if use doc; then
+		local -a HTML_DOCS=( Documentation )
+	fi
+
+	if use examples; then
+		docinto examples
+		dodoc Examples/daexutil.h
```
Comment (mgorny) [inline dev-libs/device-atlas-api-c/device-atlas-api-c-2.1.4.ebuild]: Can't you just install all the files from `Examples` directory?

File: dev-libs/device-atlas-api-c/Manifest [hunk 2]
```diff
@@ -1 +1 @@
-DIST deviceatlas-enterprise-c-2.1.3.tgz 183663 BLAKE2B e190aa8e9a4b9bc90af76e21eb425c0f040ef70e605c8f9e588bd0c859520a82807aaafddd24402d3365e94bcf3b7a875a65804f2b1b3bed6f0703f28a841440 SHA512 364811d42b2c0d937a12f2bbfee760701a1dee4e93bcc84423c8b236d59257a3440111ffc6072a7822b9694ea65986572550f67d0b1e5d68a784d1339b2b70aa
+DIST deviceatlas-enterprise-c-2.1.4.tgz 51620 SHA512 0e7fe00277715f039c661191cb865af05ff2076a6b3ccf9892fc0485fd8468a158e81ba9161df09d749540c59ba0a6c8c1730d4db4656749979d7f0c4231096f
```
Comment (mgorny) [inline dev-libs/device-atlas-api-c/Manifest]: This is in the wrong commit.

**Comment by mgorny** [comment]
Ping.

**Comment by mgorny** [comment]
Closing due to no activity.

**Comment by devnexen** [comment]
I did what was needed or is it due to maintainer's inactivity ? Thanks.

**Comment by mgorny** [comment]
No, you didn't. There are two commits stating 'version bump' and doing I-have-no-clue-what. Please take some time to learn how to create good git commits with explanatory commit messages.

**Comment by devnexen** [comment]
Ok noted ;-)

**Pull request closed without merging**